diff --git a/dist/index.mjs b/dist/index.mjs
index e7ee6225b8a3817445d4b45bffe28c04d957ffc8..8a0e49d9e080653bd023b514edbd31686de1a0bd 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1428,6 +1428,7 @@ async function transformMain(code, filename, options, pluginContext, ssr, custom
 		if (transformWithOxc) {
 			const { code, map } = await transformWithOxc(resolvedCode, filename, {
 				...options.devServer?.config.oxc,
+				tsconfig: false,
 				lang: "ts",
 				sourcemap: options.sourceMap
 			}, resolvedMap);
